run_tests.sh

```diff
@@ -3,7 +3,7 @@ set -ex
 
 dir=$(pwd)
 # list of all modules to test
-modules=". ./dex/testing/loadbot"
+modules=". /dex/testing/loadbot /client/cmd/dexc-desktop"
```

Comment:
This is odd because there are no tests in this `package main`.

Don't we want `CGO_ENABLED=1 go build -v -tags systray -o /dev/null ./client/cmd/dexc-desktop` as discussed in https://github.com/decred/dcrdex/pull/1957/files#r1173116864?

Reply:
Oh but it is a different module... unlike plain dexc.  Hmm.  I guess it forces compilation, so...